src/core_plugins/choropleth/public/choropleth_layer.js

```diff
@@ -0,0 +1,286 @@
+import $ from 'jquery';
+import L from 'leaflet';
+import _ from 'lodash';
+import d3 from 'd3';
+import { KibanaMapLayer } from 'ui/vis_maps/kibana_map_layer';
+import { truncatedColorMaps } from 'ui/vislib/components/color/truncated_colormaps';
+
+export default class ChoroplethLayer extends KibanaMapLayer {
+
+  constructor(geojsonUrl) {
+    super();
+
+    this._metrics = null;
+    this._joinField = null;
+    this._colorRamp = truncatedColorMaps['Yellow to Red'];
```

Comment:
Is this because we default to "Yellow to Red"? Is there a way to base this off the default defined in the TemplateVisType?